packages/file/fsFile-server.js

```diff
@@ -224,7 +224,108 @@ function _copyStoreData(fileObj, storeName, sourceKey, callback) {
   writeStream.once('error', function(error) {
     callback(error);
   });
-
   readStream.pipe(writeStream);
 }
 var copyStoreData = Meteor.wrapAsync(_copyStoreData);
+
+/**
+ * @method FS.File.prototype.copyData Copies the content of a store directly into another store.
+ * @public
+ * @param {string} sourceStoreName
+ * @param {string} targetStoreName
+ * @param {boolean=} move
+ */
+FS.File.prototype.copyData = function(sourceStoreName, targetStoreName, move){
+
+  move = !!move;
+  /**
+   * @type {Object.<string,*>}
+   */
+  var sourceStoreValues = this.copies[sourceStoreName];
+  /**
+   * @type {string}
+   */
+  var copyKey = cloneDataToStore(this, sourceStoreName, targetStoreName, move);
+  /**
+   * @type {Object.<string,*>}
+   */
+  var targetStoreValues = {};
+  for (var v in sourceStoreValues) {
+    if (sourceStoreValues.hasOwnProperty(v)) {
+      targetStoreValues[v] = sourceStoreValues[v]
+    }
+  }
+  targetStoreValues.key = copyKey;
+  targetStoreValues.createdAt = new Date();
+  targetStoreValues.updatedAt = new Date();
+  /**
+   *
+   * @type {modifier}
+   */
+  var modifier = {};
+  modifier.$set = {};
+  modifier.$set["copies."+targetStoreName] = targetStoreValues;
+  if(move){
+    modifier.$unset = {};
+    modifier.$unset["copies."+sourceStoreName] = "";
+  }
+  this.update(modifier);
+};
+
+// TODO maybe this should be in cfs-storage-adapter
+/**
+ *
```

Comment:
I Think you are right - this part should perhaps be in the storage adapter logic, maybe have it only take a type of storage adapter instead of string?

having a `sourceSA.copy(fileObj, destination)` and a `sourceSA.move(fileObj, destination)` ?
Just ideas - nothing frozen